lib/zhr_devs/bakery_integration/commands/command.ex

```diff
@@ -2,9 +2,14 @@ defmodule ZhrDevs.BakeryIntegration.Commands.Command do
   @moduledoc """
   Behaviour for defining commands that can be handled correctly by the CommandRunner
   """
-  @type error() :: %{error: atom(), message: term(), exit_status: nil | integer()}
+  @type error() :: %{
+          error: atom(),
+          exit_status: nil | integer(),
+          context: term(),
+          message: term()
+        }
 
-  @callback run(opts :: Keyword.t()) :: {:ok, pid()} | {:error, term()}
+  @callback run(opts :: Keyword.t()) :: {:ok, pid()} | {:error, term()} | Uptight.Result.Err.t()
```

Comment:
This is a signal that the semantics of `run` is messy.

This clearly is

```
Uptight.Result.perhaps(pid())
```

Please refactor accordingly.